infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesClientFactory.java

```diff
@@ -61,15 +62,16 @@ public KubernetesClientFactory(
           int maxConcurrentRequestsPerHost,
       @Named("che.infra.kubernetes.client.http.connection_pool.max_idle") int maxIdleConnections,
       @Named("che.infra.kubernetes.client.http.connection_pool.keep_alive_min")
-          int connectionPoolKeepAlive) {
+          int connectionPoolKeepAlive,
+      EventListener eventListener) {
```

Comment:
formatting looks a bit off here

Reply:
My offer to the god of formatting wasn't accepted. I think that is the reason why it looks like that and we have no way not to accept it. I'm sorry.